### PR TITLE
Added volume bar

### PR DESCRIPTION
**Pull request by sh1der**
still working

### CODE REVIEW

**Comment by keredson** [comment]
Ah nice.  Good call.  I'm traveling ATM; will test and merge Thursday probably. Thanks!

File: gnomecast.py [hunk 1]
```diff
@@ -641,6 +658,11 @@ def on_cast_combo_changed(self, combo):
       else:
           entry = combo.get_child()
 
+  def on_volume_changed(self, widget):
```
Comment (keredson) [inline gnomecast.py]: i'm prob. going to add a `@throttle()` here so we're not pushing too many calls.

**Comment by keredson** [comment]
this functions well but i'd like to work on the aesthetics.  

screenshot from 2018-02-23 14-56-57

perhaps there's a single speaker button where the slider currently is, and clicking brings up a popup, vertically oriented slider?

**Comment by e-ht** [comment]
@sh1der dropping in GTKVolumeButton would be give a little more polish and show purpose.
volumebutton [Reference here:](https://developer.gnome.org/gtk3/stable/GtkVolumeButton.html)
 
Also of note, I tested with a Chromecast Audio device and was not able to get the volume option to show up until I connected to a Chromecast Ultra.  From there on the volume adjustment was available for both.  If you need help testing for that you can ping me.

File: gnomecast.py [hunk 2]
```diff
@@ -584,7 +601,7 @@ def gen_thumbnail(self):
     else:
       thumbnail_fn = tempfile.mkstemp(suffix='.jpg', prefix='moviecaster_thumbnail_')[1]
       os.remove(thumbnail_fn)
-      cmd = ['ffmpeg', '-y', '-i', self.fn, '-f', 'mjpeg', '-vframes', '1', '-ss', '27', '-vf', 'scale=800:-1', thumbnail_fn]
+      cmd = ['ffmpeg', '-y', '-i', self.fn, '-f', 'mjpeg', '-vframes', '1', '-ss', '27', '-vf', 'scale=600:-1', thumbnail_fn]
```
Comment (e-ht) [inline gnomecast.py]: Why the scale change?

**Comment by keredson** [comment]
i wound up using the GTK volume button (thanks @e-ht) in https://github.com/keredson/gnomecast/commit/dc5ba3607a8d2c7a6cbed5e60a9fdf9e7c15cfb5 released in v0.2.13.  thanks both!

**Comment by sh1der** [comment]
I was working with GTKVolumeButton but I'm not that fast. I'm still learning, thanks both ;)

**Pull request closed without merging**